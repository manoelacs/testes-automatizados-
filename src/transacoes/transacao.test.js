import React from "react";
import { render } from "@testing-library/react";
import Transacao from "./Transacao";

describe("Coponente de transação do extrato", () => {
  it(" O snapshot do component deve permacer sempre o mesmo", () => {
    const { container } = render(
      <Transacao data="03/12/2021" tipo="saque" valor="20.00" />
    );
    expect(container.firstChild).toMatchSnapshot();
  });
  it("Quando eu realizo uma nova transação");
});
