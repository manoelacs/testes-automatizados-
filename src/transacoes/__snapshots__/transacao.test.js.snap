// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Coponente de transação do extrato  O snapshot do component deve permacer sempre o mesmo 1`] = `
<div
  class="transacao-container"
>
  <p>
    03/12/2021
  </p>
  <p>
    saque
  </p>
  <p>
    R$ 
    20.00
  </p>
</div>
`;
